core/src/main/scala/cats/Bifoldable.scala

```diff
@@ -0,0 +1,46 @@
+package cats
+
+/**
+ * A type class abstracting over types that give rise to two independent [[cats.Foldable]]s.
+ */
+trait Bifoldable[F[_, _]] extends Serializable { self =>
```

Comment:
Simulacrum does it automatically. We probably aren't consistent with the `F[_,_]` type classes though.